scripts/entrypoint.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 set -e
 
-# Command exec
-echo Entrypoint executing: "$@"
-exec "$@"
+if [[ -n "$@" ]]; then
```

Comment:
This is an interesting pattern for handling arguments. What purpose does it serve? Will this condition ever be true?

Reply:
My bad. That got changed once I allowed the CMD override. Forgot to commit the updated script. Should be good now.